src/Database/Mock/composer.json

```diff
@@ -0,0 +1,28 @@
+{
+    "name": "tenancy/db-driver-mock",
+    "description": "The tenancy/tenancy database driver for mocking",
+    "keywords": ["database", "tenancy", "mock"],
+    "license": "MIT",
+    "require": {
+        "tenancy/framework": "*"
+    },
+    "autoload": {
+        "psr-4": {
+            "Tenancy\\Database\\Drivers\\Mock\\": ""
+        }
+    },
+    "authors": [
+        {
+            "name": "Daniël Klabbers",
+            "email": "daniel@klabbers.email",
+            "homepage": "http://luceos.com"
+        }
```

Comment:
Missing an author.

Reply:
Won't matter if we get rid of the Mock Driver / place it into the testing package anyways.